docker-compose.yml

```diff
@@ -103,9 +103,12 @@ services:
       context: ./build/deps
       dockerfile: jaeger.Dockerfile
     container_name: ferretdb_jaeger
+    environment:
+      - COLLECTOR_OTLP_ENABLED=true
     ports:
       - 16686:16686 # UI on http://127.0.0.1:16686/
       - 14268:14268 # to receive spans from clients
```

Comment:
Do we still need that port?

Reply:
We actually don't need 14268. I can remove it